index.html

```diff
@@ -156,21 +168,25 @@ <h2 class="class-project-name">Project name goes here</h2>
       <!--The section about my self-->
       <section id="id-section-about-my-self">
 
-        <!--Introduction to the the project list-->
-        <h1 class="class-sections-titles">About</h1>
+        <div class="about-container">
```

Comment:
- [ ] Something seems to be wrong with the bullets of the list items listed in the `Languages`, `Skills`, and the `Frameworks` section: they seem to be overlapping on the text. I think you should have a look and fix it.
- [ ] You might want to consider increasing the margins to the left and right of all the components on your page. The components look so close to the left and right margins of the container compared to the Figma tempate at the moment.

**_Figma Template_**
![figma](https://user-images.githubusercontent.com/26856491/122379449-c7268d00-cf5e-11eb-93e4-298f167142b5.png)

**_Your Design_**
![design](https://user-images.githubusercontent.com/26856491/122379459-caba1400-cf5e-11eb-9433-64df19a3f904.png)